lib/elixir/lib/task.ex

```diff
@@ -12,46 +12,46 @@ defmodule Task do
       res  = do_some_other_work()
       res + Task.await(task)
 
-  Tasks spawned with `async` can be waited on by their caller
+  Tasks spawned with `async` can be awaited on by their caller
```

Comment:
is the correct english is "awaited on by their *calling* process (and only *the process that called it*)"?

at least it sounds better to my ears, but maybe it's called the 'caller' somewhere else.